resources/js/components/NavbarProfile.vue

```diff
@@ -1,47 +1,49 @@
 <template>
   <div id="userMenu">
-    
-    
-    <a
-      data-toggle="dropdown"
-      href="#"
-      aria-expanded="false"
-      id="profileMenu"
-    ><avatar-image
-      id="avatarMenu"
-      class-container="d-flex m-1"
-      size="40"
-      class-image="m-1"
-      :input-data="information"
-    ></avatar-image></a>
+    <a data-toggle="dropdown" href="#" aria-expanded="false" id="profileMenu">
+      <avatar-image
+        id="avatarMenu"
+        class-container="d-flex m-1"
+        size="40"
+        class-image="m-1"
+        :input-data="information"
+      ></avatar-image>
+    </a>
 
-    <b-popover
-      target="profileMenu"
-      triggers="click blur"
-      placement="bottomleft"
-      container="userMenu"
-      ref="popover"
-      @hidden="onHidden"
-    >
-      <template slot="title">
-        <div class="wrap-name">{{fullName}}</div>
-      </template>
+    <b-popover target="profileMenu" placement="bottomleft" triggers="click blur">
       <template>
-        <a
-          data-v-2eb90a9e
-          href="https://processmaker.gitbook.io/processmaker/"
-          class="dropdown-item item"
-          target="_blank"
-        >
-          <i data-v-2eb90a9e class="fas fa-question-circle fa-fw fa-lg"></i>
-          {{$t('Help')}}
-        </a>
-        <template v-for="item in items">
-          <a class="dropdown-item item" :href="item.url">
-            <i :class="item.class"></i>
-            {{item.title}}
-          </a>
-        </template>
+        <ul class="list-group list-group-flush m-1">
+          <li class="list-group-item px-1">
+            <a :href="user_id">
+              <i class="fas fa-user fa-fw fa-lg"></i>
+              {{$t('View Profile')}}
+            </a>
+          </li>
+          <li class="list-group-item px-1">
+            <a href="/profile/edit">
+              <i class="fas fa-user-cog fa-fw fa-lg"></i>
+              {{$t('Profile Settings')}}
```

Comment:
Recommend we change this to "Edit Profile" to better pair with "View Profile" as well as the breadcrumb navigation.